houses/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+class House(models.Model):
+    name        = models.CharField(max_length=100)
+    description = models.TextField()
+    latitude    = models.DecimalField(max_digits=16, decimal_places=14, default=0.0)
+    longitude   = models.DecimalField(max_digits=17, decimal_places=14, default=0.0)
+    max_guest   = models.IntegerField()
+    trap        = models.BooleanField()
```

Comment:
`BooleanField()`를 사용하는 경우는 데이터가 0아니면 1입니다.
그렇기 때문에 default 값을 True/False로 선언해주시는게 좋겠습니다-!
```suggestion
    trap        = models.BooleanField(default=False)
```

Reply:
수정했습니다 😄